tests/runtime/delete_wildcard

```diff
@@ -0,0 +1,14 @@
+NAME delete_wildcard_idx_0
+PROG BEGIN { @map[10, 10] = 1; @map[2, 20] = 2; @map[10, 11] = 1; delete(@map[10, *]); print(@map); exit(); } END {}
+EXPECT @map\[2, 20\]: 2
```

Comment:
This doesn't actually check that the other values are gone now, right? Cuz this would be on its own line. To check that the other entries are deleted, you'd want to probably use json output here.

Reply:
That's correct. We could either use JSON output or I'm thinking about adding a new `EXPECT_NOT` clause to runtime tests to specify regex which should not be present in the output. But using JSON is probably better.